Source/Core/OrientedBoundingBox.js

```diff
@@ -419,6 +419,34 @@ define([
         return result;
     };
 
+    /**
+     * @private
+     */
+    OrientedBoundingBox.closestLocationIn = function(position, boundingObject) {
+        if (defined(boundingObject)) {
+            // to avoid inverse of a zero matrix (since that doesn't exist).
+            if (this.boundingObject.halfAxes.equals(Matrix3.ZERO)) {
+                position = new Cartesian3();
+                return position;
+            }
+
+            // convert world space positionToCheck to orientedBoundingBox's object space.
+            var inverseTransformationMatrix = Matrix3.inverse(boundingObject.halfAxes, new Matrix3.fromScale(1.0));
```

Comment:
`new Matrix3.fromScale(1.0)` -> `Matrix3.fromScale`

Also, functions like this are creating a `new Matrix3` if you don't define the result parameter.  For cases like this, we usually define a scratch variable outside the function scope.  Here is an example: 

https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/PolygonGeometry.js#L60
https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/PolygonGeometry.js#L64

Doing things like this prevent Cesium from slowing down from having to garbage collect a bunch of objects.